cloudformation_templates/aws_nginx.json

```diff
@@ -288,9 +288,17 @@
         "AvailabilityZones" : {"Fn::GetAZs": ""},
         "LaunchConfigurationName" : {"Ref": "LaunchConfiguration"},
         "LoadBalancerNames": [{"Ref": "LoadBalancer"}],
+        "HealthCheckType": "ELB",
+        "HealthCheckGracePeriod": 300,
```

Comment:
Yes, since it should update instances one by one this will act as an additional delay for each instance.

I was trying to figure out a bit more about this as it's not clear when the timer starts (is it the instance start time or the time between EC2 health check and ELB one, since default EC2 health check seemed to work fine without it etc.). But most things I found recommend setting it to a high value, mostly because otherwise ASG can fail to bring up new instances and just restart them over and over again.